js.js

```diff
@@ -9,4 +9,6 @@ for(i=0; i<101; i++) {
     else {
```

Comment:
if the number is divisible by both 5 and 3 you should print "fizz buzz"

Reply:
Added the feature in the code pushed